Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -3604,6 +3605,23 @@ namespace AzToolsFramework
         m_selectedEntityIds.clear();
         m_selectedEntityIds.reserve(selectedEntityIds.size());
         AZStd::copy(selectedEntityIds.begin(), selectedEntityIds.end(), AZStd::inserter(m_selectedEntityIds, m_selectedEntityIds.end()));
+
+        // Test - remove entities that cannot be edited
+        // This is not the right place for this logic most likely, just testing if it would work...
```

Comment:
question: Should this code and/or comment be removed?

Reply:
Oh yes, I wrote this before I tried all other options and ended up reverting to this for now :'D